ReactLearning/src/App.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import './App.css';
+import LoginPage from './page/Login/index';
```

Comment:
No need to mention index, it automatically imports index.js file. There must be only one index.js file inside a folder.